docker-compose-fullstack.yml

```diff
@@ -0,0 +1,43 @@
+version: '3'
+
+services:
```

Comment:
Since the intention is just to run the app, we can not expose the ports to the host and only expose them to the `website` service.